consensus/ibft/src/main/java/org/hyperledger/besu/consensus/ibft/messagewrappers/RoundChange.java

```diff
@@ -73,7 +76,12 @@ public static RoundChange decode(final Bytes data) {
         SignedData.readSignedRoundChangePayloadFrom(rlpIn);
     Optional<Block> block = Optional.empty();
     if (!rlpIn.nextIsNull()) {
-      block = Optional.of(Block.readFrom(rlpIn, IbftBlockHeaderFunctions.forCommittedSeal()));
+      block =
+          Optional.of(
+              ProtocolRLPSpec.decodeBlockStandalone(
+                  IbftProtocolSchedule.create(new StubGenesisConfigOptions()),
```

Comment:
What if the IBFT network forks between versions that have typed transactions and those that don't?  If we are passing in the same freshly created schedule then do we need to pass it in?  Or create it every time?